stage_tester/dev.conf

```diff
@@ -0,0 +1,8 @@
+[load_test]
+
+offline_token=..
+login_url=https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
+client_id=rhsm-api
+api_url=https://console.redhat.com
+organization=
+log_level=error
```

Comment:
This file shouldn't be in the sources directory. Maybe you can have a `config` directory in the repository root or just leave the `dev.conf` in the repository root